src/graphics/context.rs

```diff
@@ -113,10 +113,14 @@ impl GraphicsContext {
             })
         };
 
+        let mut base_limits = wgpu::Limits::default();
         if conf.backend == Backend::All {
+            base_limits = wgpu::Limits::downlevel_webgl2_defaults();
+
```

Comment:
That is actually intended, because `Backend::All` **MAY** also include the OpenGL and therefore WebGL backend. The WebGL backend strictly does not support compute shaders (that's why they are explicitly disabled in the `Limits::downlevel_webgl2_defaults()` limits).

So a user may write their project using `Backend::All`, may use compute shaders in that project, and it will probably work fine. But once the `Backend::All` uses the OpenGL backend, the project may suddenly critically fail.